airbyte-webapp/src/components/connectorBuilder/Builder/StreamConfigView.module.scss

```diff
@@ -0,0 +1,37 @@
+@use "scss/variables";
+@use "scss/colors";
+
+$deleteButtonWidth: 24px;
+
+.container {
```

Comment:
Seems like the container and heading stuff can be pulled into a general place - I need the same stuff for the inputs

Reply:
Consolidatd these styles into a shared `BuilderConfigView` component